controller/konnect/ops/ops.go

```diff
@@ -511,6 +542,15 @@ func logOpComplete[
 	T constraints.SupportedKonnectEntityType,
 	TEnt constraints.EntityType[T],
 ](ctx context.Context, start time.Time, op Op, e TEnt, err error) {
+
+	// if the entity is a Mirror KonnectGatewayControlPlane, don't log the konnect operation,
+	// as no operation occurred.
+	if cp, ok := any(e).(*konnectv1alpha1.KonnectGatewayControlPlane); ok {
+		if isMirrorControlPlane(cp) {
+			return
+		}
+	}
```

Comment:
It would be nice to add a function which checks `isMirror(e)` and returns `bool`.